test-apps/display-performance-test-app/README.md

```diff
@@ -83,7 +87,7 @@ The default configuration file allows you to specify the following:
 You can specify filename options that you wish to ignore:
 When you chose to produce images through an 'image' or 'both' test, the program will name them in the following format: modelName_viewName_renderMode_viewFlagOpts_renderModeOpts_tilePropsOpts.png (for example, SmallTextTest008_Wireframe_-fll+scL+cmL+slL-clp+con_+solShd_+inst.png). If you chose to do a 'readPixels' test, the program will save the images produced from this in the following format: depth/elemId/type_readPixelsSelectorOpts_modelName_viewName_renderMode_viewFlagOpts_renderModeOpts_tilePropsOpts.png (for example, type_+geom+dist_TimingTest01_V0_HiddenLine_-fll+scL+cmL+slL-clp+con_+solShd_+inst.png). The read pixels selector options, view flag options, render mode options, and tile properties options will all be an abbreviated string representation of the actual property. For example, the render mode option 'enableInstancing' would be shown as '+inst' in the filename. The 'filenameOptsToIgnore' property allows you to specify either an array of strings or a single space delimited string, where each string item is an abbreviated string representation of a read pixels selector option, view flag option, render mode option, or tile properties option. These options will not be included in the image names.
 
-You can specify which saved view you wish to use for each test with the viewName property.  If external saved views exist for the local file you can specify which external saved view to use with the extViewName property (you can also specify external saved views by name using the viewName property if the name does not clash with a normal saved view).  You can also specify a saved view to use via the viewString property.  The viewString property must include the _viewname and_viewStatePropsString properties (and optionally the _selectedElements and/or_overrideElements properties) which you can copy from the external saved view file.
+You can specify which saved view you wish to use for each test with the viewName property.  If external saved views exist for the local file you can specify which external saved view to use with the extViewName property (you can also specify external saved views by name using the viewName property if the name does not clash with a normal saved view).  You can also specify a saved view to use via the viewString property.  The viewString property must include the _viewname_ and _viewStatePropsString_ properties (and optionally the _selectedElements_ and/or _overrideElements_ properties) which you can copy from the external saved view file.
```

Comment:
Did your editor append underscores to these property names in a mistaken assumption you were trying to underline them?

Reply:
No, I added them since they did not look right in my preview window with the underscores that were there, and I assumed they were attempting to be italicized.

Reply:
The property names begin with an underscore. If you want to make that clearer you can surround them with backticks.

Reply:
Okay, I guess what threw me was the lack of spaces before several of the underscores.
But shouldn't _viewname be _name?

Reply:
Done, assuming I was correct about _name instead of _viewname.